npm/qsharp/src/language-service/language-service.ts

```diff
@@ -274,6 +285,18 @@ export class QSharpLanguageService implements ILanguageService {
       log.error("Error in onDiagnostics", e);
     }
   }
+
+  async onTestCallables(callables: [string, ILocation][]) {
```

Comment:
As mentioned in the WASM layer, I think this type can be `ITestCallables` instead of a tuple, the reasoning being that objects are just easier to use/read in JavaScript